query-node/substrate-query-framework/index-builder/src/bootstrap/Bootstrapper.ts

```diff
@@ -89,13 +88,13 @@ export default class Bootstrapper {
      * bootstrapped the data
      * 
      * @param em  `EntityManager` by `typeorm`
-     * @param boot boothandler
+     * @param bootEvent SubstrateEvent
      */
-    private async shouldBootstrap(em: EntityManager, boot: BootstrapFunc):Promise<boolean> {
+    private async shouldBootstrap(em: EntityManager, bootEvent: SubstrateEvent):Promise<boolean> {
         const event = await em.findOne(SavedEntityEvent, { 
             where: { 
-                event_name: 'Bootstrap', 
-                event_method: boot.name
+                eventName: bootEvent.event_name, 
```

Comment:
Bootstrap events are not "real" events. So `event_name` is always `Bootstrap` and `event_method` simply indicates the executed handler. It is enough to locate the event and decide is the bootstrap was successfull

Reply:
That makes sense, but we don't have `event_method` field on `SavedEntityEvent`.

Reply:
Oh okay, so it's been refactored out at some point? So I suggest using eventName: `Bootstrap.${boot.name}` then and lookup by the name only.